app/actualize_script.php

```diff
@@ -72,6 +72,9 @@ function notice($message) {
 	}
 
 	FreshRSS_Auth::giveAccess();
+
+	Minz_ExtensionManager::callHook('freshrss_user_maintenance');
```

Comment:
Would be good to put it in the CLI version as well https://github.com/FreshRSS/FreshRSS/blob/master/cli/actualize-user.php

Reply:
I've tried but it seems that it is not triggered. :/

Reply:
Cou you please try with https://github.com/FreshRSS/FreshRSS/pull/3443 ?

Reply:
I've tried without success

Reply:
Have you tested with a system or user extension?

Reply:
Could you please try again with https://github.com/FreshRSS/FreshRSS/pull/3443 ?

Reply:
I tried with a system extension

Reply:
So far, no success either :/

Reply:
Let's merge here and come back to it when https://github.com/FreshRSS/FreshRSS/pull/3443 works